### ISSUE TITLE
Is this a fork from https://github.com/rubensousa/RecyclerViewSnap ?

### ISSUE THREAD

**Issue by andremion** [DevExchanges/SnappingRecyclerview#1]
Is this a fork from @rubensousa 's [RecyclerViewSnap](https://github.com/rubensousa/RecyclerViewSnap) ?

**Comment by DevExchanges** [comment]
Yes, andremion! I get **GravitySnapHelper.java** from this project! :dancer:

**Comment by rubensousa** [comment]
You should include the Apache-2 license then. I noticed you wrote an article on: http://www.devexchanges.info/2016/09/android-tip-recyclerview-snapping-with.html

Thanks for not giving credit to my work.

**Comment by JoaquimLey** [comment]
I don't get it, why would someone try to take credit from someone else's **open source** work, sad.

**Comment by DevExchanges** [comment]
Very sorry, rubensousa! 
I've just updated Apache-2 Licence and make reference to your original file on your project on my article!
Thanks for reminding me!

**Comment by JoaquimLey** [comment]
Btw you also forgot that changing the ordering of the methods and making small changes isn't enough to **not get caught**.
